tagmanager/src/tm_workspace.c

```diff
@@ -409,14 +413,21 @@ static void write_includes_file(FILE *fp, GList *includes_files)
 		g_free(str);
 		node = g_list_next(node);
 	}
+
+	fclose(fp);
+
+	return TRUE;
 }
 
 
-static void append_to_temp_file(FILE *fp, GList *file_list)
+static gboolean append_to_temp_file(const gchar *outf, GList *file_list)
```

Comment:
that function name is now a lie: it doesn't append anymore, but simply replaces the file's content.